packages/types/src/package.ts

```diff
@@ -46,6 +46,10 @@ export interface PeerDependenciesMeta {
   }
 }
 
+export interface PackagePublishConfig extends Record<string, unknown> {
```

Comment:
I guess PublishConfig should be enough

Also, add a changeset for `@pnpm/types`